.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.watcherExclude": {
+        "**/target": true
+    }
+}
```

Comment:
Can you remove this, I think it was added by accident.

Reply:
Sorry, I accidently introduced the file, removed